pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,8 +1,28 @@
+# construct an executable file that wraps the actual executable
```

Comment:
shouldn't documentation like this be kept outside of the scripts to prevent a mass-rebuild? See https://github.com/NixOS/nixpkgs/pull/16173